Bottom: 325a433cc5b2e8ac7bb39b0fb55adeb76e6b8a2a
Top:    8a1726574ce1fa269490cf146ad50baa4ba6b83f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:48:58 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c19a0ba..e252ff1 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1296,6 +1296,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
